mod/admin.php

```diff
@@ -615,6 +615,15 @@ function admin_page_summary(App $a) {
 		$showwarning = true;
 		$warningtext[] = sprintf(t('Your DB still runs with MyISAM tables. You should change the engine type to InnoDB. As Friendica will use InnoDB only features in the future, you should change this! See <a href="%s">here</a> for a guide that may be helpful converting the table engines. You may also use the command <tt>php include/dbstructure.php toinnodb</tt> of your Friendica installation for an automatic conversion.<br />'), 'https://dev.mysql.com/doc/refman/5.7/en/converting-tables-to-innodb.html');
 	}
+	// Check if github.com/friendica/master/VERSION is higher then
+	// the local version of Friendica. Check is opt-in, source may be master or devel branch
+	if (Config::get('system', 'check_new_version_url', 'none') != 'none' ) {
+		$gitversion = Config::get('system','git_friendica_version');
+		if (version_compare(FRIENDICA_VERSION, $gitversion)<0) {
+			$warningtext[] = t('There is a new version of Friendica available for download.');
```

Comment:
Can you please remind the admin of their current version and the new version in the message since we have both information available?